string device_path
---
bool success
string message
